### PR TITLE
Dashboard visuals

### PR DESCRIPTION
**Pull request by david-nepo**
**main.py**
- streamlit dashboard runs
- requires 'mock_data_multi_plants.csv' (which is created from create_mock_data.py'
- TODO: add database connection functionality
- TODO: abstract further into functions

**create_mock_data.py**
- extrapolates one reading from 6 plants to emulate 24-hr dataset
- outputs 'mock_data_multi_plants.csv' 

**playground.ipynb**
- notebook for experimenting with chart design
- no outputs

### CODE REVIEW

File: dashboard/main.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+import pandas as pd
+import altair as alt
+import streamlit as st
+from datetime import timedelta
+
+
+RESAMPLE_RATE = '10T'
+LINE_WIDTH = 2
+FIG_WIDTH = 1200
```
Comment (caitlinturnidge) [inline dashboard/main.py:11]: I like these constants!

File: dashboard/main.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        # titleFontSize=AXIS_LABEL_FONT_SIZE,
+    ).configure_title(
+        # fontSize=TITLE_FONT_SIZE,
+    )
+
+    return chart
+
+
+if __name__ == "__main__":
+
```
Comment (caitlinturnidge) [inline dashboard/main.py:86]: As you have already said, can definitely split this into functions

Comment (david-nepo) [reply dashboard/main.py:86]: will do!

**Review by caitlinturnidge — APPROVED** [review]
Have seen the dashboard and I think it is great!

**Pull request merged**